Code/ForceField/AngleConstraint.cpp

```diff
@@ -52,15 +52,14 @@ AngleConstraintContrib::AngleConstraintContrib(
 
   double angle = 0.0;
```

Comment:
This can move inside the `if(relative)` loop if you make the other suggested change below

Reply:
yes, if the `+=` go inside, then this is unnecessary and we can just define it when we calculate it.